src/main/java/com/aws/greengrass/logmanager/model/ComponentLogConfiguration.java

```diff
@@ -25,6 +25,5 @@ public class ComponentLogConfiguration {
     private Level minimumLogLevel = Level.INFO;
     private Long diskSpaceLimit;
     private boolean deleteLogFileAfterCloudUpload;
-    private boolean uploadToCloudWatch;
```

Comment:
why are we removing this option?

Reply:
This is wrongly added before, which should be in the next PR to allow userComponent has the boolean value.